### PR TITLE
separated related code, added folders for stylesheets and js code, fi…

### PR DESCRIPTION
**Pull request by christinachance**
…xed deletion

### CODE REVIEW

File: portfolio/src/main/webapp/scripts/blog_script.js [hunk 1]
```diff
@@ -66,19 +66,22 @@ function createCommentElement(comment, currUserEmail, isAdmin){
...
 }
 
-function deleteComment(comment){
-    const params = new URLSearchParams();
-    params.append('id', comment.id);
-    fetch('/delete-comment', {method: 'POST', body:params});
+async function deleteComment(comment){
+    (async () =>{
+        const params = new URLSearchParams();
+        console.log(comment.commentId);
```
Comment (wulian1223) [inline portfolio/src/main/webapp/scripts/blog_script.js:72]: delete debugging code.

File: portfolio/src/main/webapp/scripts/blog_script.js [hunk 2]
```diff
@@ -66,19 +66,22 @@ function createCommentElement(comment, currUserEmail, isAdmin){
     return commentElement;
 }
 
-function deleteComment(comment){
-    const params = new URLSearchParams();
-    params.append('id', comment.id);
-    fetch('/delete-comment', {method: 'POST', body:params});
+async function deleteComment(comment){
+    (async () =>{
```
Comment (wulian1223) [inline portfolio/src/main/webapp/scripts/blog_script.js:70]: Any reason to use IIFE? I feel it is same as 
```
function deleteComment(comment){
    const params = new URLSearchParams();
    params.append('commentId', comment.commentId);
    fetch('/delete-comment', {method: 'POST', body:params});
}
```

File: portfolio/src/main/webapp/scripts/blog_script.js (hunk 1 above)
Comment (christinachance) [reply portfolio/src/main/webapp/scripts/blog_script.js:72]: done.

File: portfolio/src/main/webapp/scripts/blog_script.js (hunk 2 above)
Comment (christinachance) [reply portfolio/src/main/webapp/scripts/blog_script.js:70]: Before using it, the code wasn't actually deleting my comments from the datastore, only the page view.

Comment (wulian1223) [reply portfolio/src/main/webapp/scripts/blog_script.js:73]: Just FYI, you could use try
```async function deleteComment(comment){
    const params = new URLSearchParams();
    params.append('commentId', comment.commentId);
    await fetch('/delete-comment', {method: 'POST', body:params});
}
```
`await` will block until the fetch gets done.

**Pull request merged**